build.rs

```diff
@@ -453,6 +453,7 @@ fn generate_bindings(r_paths: &InstallationPaths, version_info: &RVersionInfo) {
     // the resulting bindings.
     let mut bindgen_builder = bindgen::Builder::default()
         .emit_diagnostics()
+        .layout_tests(false)
```

Comment:
Can we make it switchable? I definitely want to run layout tests when building libR-sys from scratch.

Reply:
Could you explain more? The only time this has come up, was when bindgen was generating the layout tests wrong for some packed-something.
Is this for our CI benefit? For libR-sys developers?

Reply:
Alright, I think I did it now! It just made this PR a little more complicated.